### PR TITLE
Feature/alerts reported color

### PR DESCRIPTION
**Pull request by j8seangel**
Store the long + lat alert position in each report sent to show them later in the map with another color.

Also fix the package versions to avoid breaks in the package-lock.json.

### CODE REVIEW

File: app/components/map/clusters/index.js [hunk 1]
```diff
@@ -14,27 +12,40 @@ class Clusters extends PureComponent {
...
+                zoomTo={this.props.zoomTo}
+                datasetSlug={this.props.datasetSlug}
+              />
+            );
+          }
+          let markerColor = styles.gladColor;
+          const id = `${marker.geometry.coordinates[0]}${marker.geometry.coordinates[1]}`;
+          if (this.props.reportedAlerts.includes(id)) {
+            markerColor = styles.reportedColor;
+          } else if (datasetSlug === 'viirs') {
```
Comment (sorodrigo) [inline app/components/map/clusters/index.js]: I don't get this. If not reported then viirs? shouldn't we check first for datasetSlug and afterwards for reported or not?

File: app/config/db/schemas.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const AlertSchema = {
```
Comment (sorodrigo) [inline app/config/db/schemas.js]: noice

File: app/redux-modules/alerts.js [hunk 3]
```diff
@@ -54,6 +56,17 @@ export default function reducer(state = initialState, action) {
       return { ...state, canDisplayAlerts: action.payload };
     case SET_ACTIVE_ALERTS:
       return { ...state, clusters: action.payload };
+    case UPLOAD_REPORT_COMMIT: {
+      const { alerts } = action.payload;
+      let reported = { ...state.reported };
+
+      if (alerts && alerts.length) {
+        reported.forEach((alert) => {
+          reported = [...reported, `${alert.long}${alert.lat}`];
```
Comment (sorodrigo) [inline app/redux-modules/alerts.js]: Loving this approach

File: app/components/map/clusters/index.js (hunk 1 above)
Comment (j8seangel) [inline app/components/map/clusters/index.js]: It is the same, but I agree! I've just changed.

**Pull request merged**